integration-tests/cli/csv-parser-price.it.js

```diff
@@ -55,8 +55,7 @@ describe('CSV and CLI Tests', () => {
       })
     })
 
-    // FIXME: fails with `ERR! No type with key \'custom-type\' found`
-    test.skip('should take input from file', (done) => {
+    test('should take input from file', (done) => {
```

Comment:
I just fixed all tests in this PR... Okay then, I'd create a separete PR for this 👍🏾